diff --git a/node_modules/@aws-amplify/auth/lib-esm/Auth.js b/node_modules/@aws-amplify/auth/lib-esm/Auth.js
index f095f26..2470f34 100644
--- a/node_modules/@aws-amplify/auth/lib-esm/Auth.js
+++ b/node_modules/@aws-amplify/auth/lib-esm/Auth.js
@@ -4,7 +4,7 @@ import { __assign, __awaiter, __generator, __read } from "tslib";
 import { isUsernamePasswordOpts, isCognitoHostedOpts, isFederatedSignInOptions, isFederatedSignInOptionsCustom, hasCustomState, } from './types';
 import { Amplify, ConsoleLogger as Logger, Credentials, Hub, StorageHelper, Platform, browserOrNode, parseAWSExports, UniversalStorage, urlSafeDecode, } from '@aws-amplify/core';
 import { CookieStorage, CognitoUserPool, AuthenticationDetails, CognitoUser, CognitoUserSession, CognitoUserAttribute, CognitoIdToken, CognitoRefreshToken, CognitoAccessToken, } from 'amazon-cognito-identity-js';
-import { addAuthCategoryToCognitoUserAgent, addFrameworkToCognitoUserAgent, } from 'amazon-cognito-identity-js/internals';
+import { addAuthCategoryToCognitoUserAgent, addFrameworkToCognitoUserAgent, } from 'amazon-cognito-identity-js/lib/internals/index.js';
 import { parse } from 'url';
 import OAuth from './OAuth/OAuth';
 import { default as urlListener } from './urlListener';
diff --git a/node_modules/@aws-amplify/auth/lib/Auth.js b/node_modules/@aws-amplify/auth/lib/Auth.js
index c84980d..d3e0107 100644
--- a/node_modules/@aws-amplify/auth/lib/Auth.js
+++ b/node_modules/@aws-amplify/auth/lib/Auth.js
@@ -6,7 +6,7 @@ var tslib_1 = require("tslib");
 var types_1 = require("./types");
 var core_1 = require("@aws-amplify/core");
 var amazon_cognito_identity_js_1 = require("amazon-cognito-identity-js");
-var internals_1 = require("amazon-cognito-identity-js/internals");
+var internals_1 = require("amazon-cognito-identity-js/lib/internals/index.js");
 var url_1 = require("url");
 var OAuth_1 = tslib_1.__importDefault(require("./OAuth/OAuth"));
 var urlListener_1 = tslib_1.__importDefault(require("./urlListener"));
