diff --git a/node_modules/amazon-cognito-identity-js/es/CognitoUser.js b/node_modules/amazon-cognito-identity-js/es/CognitoUser.js
index b7cf59e..18ea789 100644
--- a/node_modules/amazon-cognito-identity-js/es/CognitoUser.js
+++ b/node_modules/amazon-cognito-identity-js/es/CognitoUser.js
@@ -5,7 +5,7 @@
 
 import { Buffer } from 'buffer';
 import { Sha256 } from '@aws-crypto/sha256-js';
-import { Platform } from './Platform';
+import { Platform } from './Platform/index.js';
 import BigInteger from './BigInteger';
 import AuthenticationHelper from './AuthenticationHelper';
 import CognitoAccessToken from './CognitoAccessToken';
diff --git a/node_modules/amazon-cognito-identity-js/es/UserAgent.js b/node_modules/amazon-cognito-identity-js/es/UserAgent.js
index e41546b..64f2fbd 100644
--- a/node_modules/amazon-cognito-identity-js/es/UserAgent.js
+++ b/node_modules/amazon-cognito-identity-js/es/UserAgent.js
@@ -1,4 +1,4 @@
-import { getUserAgent } from './Platform';
+import { getUserAgent } from './Platform/index.js';
 import { AUTH_CATEGORY } from './Platform/constants';
 
 // constructor
diff --git a/node_modules/amazon-cognito-identity-js/lib/CognitoUser.js b/node_modules/amazon-cognito-identity-js/lib/CognitoUser.js
index 06b9029..4c1e8f7 100644
--- a/node_modules/amazon-cognito-identity-js/lib/CognitoUser.js
+++ b/node_modules/amazon-cognito-identity-js/lib/CognitoUser.js
@@ -4,7 +4,7 @@ exports.__esModule = true;
 exports["default"] = void 0;
 var _buffer = require("buffer");
 var _sha256Js = require("@aws-crypto/sha256-js");
-var _Platform = require("./Platform");
+var _Platform = require("./Platform/index.js");
 var _BigInteger = _interopRequireDefault(require("./BigInteger"));
 var _AuthenticationHelper = _interopRequireDefault(require("./AuthenticationHelper"));
 var _CognitoAccessToken = _interopRequireDefault(require("./CognitoAccessToken"));
diff --git a/node_modules/amazon-cognito-identity-js/lib/UserAgent.js b/node_modules/amazon-cognito-identity-js/lib/UserAgent.js
index 027fc18..ba75832 100644
--- a/node_modules/amazon-cognito-identity-js/lib/UserAgent.js
+++ b/node_modules/amazon-cognito-identity-js/lib/UserAgent.js
@@ -2,7 +2,7 @@
 
 exports.__esModule = true;
 exports.getAmplifyUserAgent = exports["default"] = exports.appendToCognitoUserAgent = exports.addFrameworkToCognitoUserAgent = exports.addAuthCategoryToCognitoUserAgent = void 0;
-var _Platform = require("./Platform");
+var _Platform = require("./Platform/index.js");
 var _constants = require("./Platform/constants");
 // constructor
 function UserAgent() {}
